src/containers/Clusters/columns.tsx

```diff
@@ -40,36 +55,75 @@ export const CLUSTERS_COLUMNS: Column<PreparedCluster>[] = [
+                if (isDeleteClusterAvailable) {
+                    menuItems.push({
+                        text: i18n('remove-cluster'),
+                        iconStart: <TrashBin />,
+                        action: () => {
+                            uiFactory.onDeleteCluster?.({
+                                clusterData: row,
+                            });
+                        },
+                        className: b('remove-cluster'),
+                    });
+                }
+
+                if (!menuItems.length) {
+                    return null;
+                }
+
+                return <DropdownMenu items={menuItems} />;
+            };
+
             return (
-                <div className={b('cluster')}>
-                    {clusterStatus ? (
-                        <ExternalLink href={clusterPath}>
-                            <div
-                                className={b('cluster-status', {
-                                    type: clusterStatus && clusterStatus.toLowerCase(),
-                                })}
-                            />
-                        </ExternalLink>
-                    ) : (
-                        <div className={b('cluster-status')}>
-                            <HelpPopover
-                                content={
-                                    <span className={b('tooltip-content')}>
-                                        {row.cluster?.error || i18n('tooltip_no-cluster-data')}
-                                    </span>
-                                }
-                                offset={{left: 0}}
-                            />
+                <Flex alignItems={'center'} justifyContent={'space-between'}>
+                    <Flex alignItems={'center'}>
+                        {clusterStatus ? (
+                            <ExternalLink href={clusterPath}>
+                                <div
+                                    className={b('cluster-status', {
+                                        type: clusterStatus && clusterStatus.toLowerCase(),
+                                    })}
+                                />
+                            </ExternalLink>
+                        ) : (
+                            <div className={b('cluster-status')}>
+                                <HelpPopover
+                                    content={
+                                        <span className={b('tooltip-content')}>
+                                            {row.cluster?.error || i18n('tooltip_no-cluster-data')}
+                                        </span>
+                                    }
+                                    offset={{left: 0}}
+                                />
+                            </div>
+                        )}
+                        <div className={b('cluster-name')}>
+                            <ExternalLink href={clusterPath}>{row.title ?? row.name}</ExternalLink>
```

Comment:
Use `name` when there is no `title`

<img width="848" alt="Screenshot 2025-06-04 at 20 55 53" src="https://github.com/user-attachments/assets/15b115d4-1321-4ce1-b28e-4f1392d5be5b" />